src/promise-katas-1.js

```diff
@@ -33,71 +33,118 @@ Remember, a Promise can only resolve successfully, or reject when it fails to re
 // when we call returnPromise like 'returnPromise()' we expect our function to use the
 // 'resolve' Promise callback function in order to resolve successfully
 
-const returnPromise = () => { };
+const returnPromise = () => {
+  return new Promise((resolve) => {
+    const promiseFunc = () => {};
+    resolve(promiseFunc)
+  })
+};
 
 // 2 Create a function that returns a promise which returns 10 if it resolves succesfully
 // Example:
 // returnTen() resolves to the value 10
 
-const returnTen = () => { };
+const returnTen = () => {
+  return new Promise((resolve) => {
+   const returnTen = 10;
+    resolve(returnTen)
+  })
+ };
 
 // 3 Create a function that returns a promise which returns 'string' if it resolves succesfully
 // Example:
 // returnString() resolves to the value 'string'
 
-const returnString = () => { };
+const returnString = () => { 
+  return new Promise((resolve) => {
+    const returnsString = 'string';
+    resolve(returnsString)
+  })
+};
 
 // 4 Create a function that returns a promise which returns an object with a key of name and value of 'Bob' if it resolves succesfully
 // Example:
 // returnBob() resolves to the value { name: 'Bob' }
 
-const returnBob = () => { };
+const returnBob = () => { 
+  return new Promise ((resolve) => {
+    const returnBob = { name: 'Bob'};
+    resolve(returnBob);
+  })
+};
 
 // 5 Create a function that returns a promise which returns an array with 4 strings of 'eggs', 'apples', 'milk' and 'bread' if it resolves succesfully
 // Example:
 // returnList() resolves to the value ['eggs', 'apples', 'milk', 'bread']
 
-const returnList = () => { };
+const returnList = () => {
+  return new Promise((resolve) => {
+    const returnList = ['eggs', 'apples', 'milk', 'bread'];
+    resolve(returnList);
+  })
+ };
 
 // 6 Create a function that returns a promise which returns "An error occurred" if it rejects
 // Example:
 // anError() rejects to "An error occurred"
 
-const anError = () => { };
+const anError = () => { 
+  return Promise.reject("An error occurred");
```

Comment:
This is just missing a couple bits:
````
return new Promise((_, reject) => {
    reject("An error occurred");
  });
`````
Have to return a new Promise which takes in a resolve and a reject, as there wont be resolve returned you can just swap it with _